ts/src/phemex.ts

```diff
@@ -2381,6 +2385,56 @@ export default class phemex extends Exchange {
                 const triggerType = this.safeString (params, 'triggerType', 'ByMarkPrice');
                 request['triggerType'] = triggerType;
             }
+            if (stopLossDefined || takeProfitDefined) {
+                if (stopLossDefined) {
+                    const stopLossTriggerPrice = this.safeValue (stopLoss, 'triggerPrice');
+                    if (stopLossTriggerPrice === undefined) {
+                        throw new InvalidOrder (this.id + ' createOrder() requires a trigger price in params["stopLoss"]["triggerPrice"], or params["stopLoss"]["stopPrice"], or params["stopLoss"]["stopLossPrice"] for a stop loss order');
```

Comment:
@AlgoQ this error message does not seem to match the reality because right now, you're only checking `triggerPrice` inside `stopLoss` so `'stopLoss: { stopLossPrice/stopPrice/etc: yy}```` wouldn't work. Btw I think `tringgerPrice/stopPrice support would be enough because we're already inside the `stopLoss/takeProfit` object so we don't need to repeat that part